test/providers/store/fileScan.js

```diff
@@ -19,10 +19,10 @@ describe('list a tool result ', () => {
       if (path.includes('error')) throw new Error('test error')
       const result = [
         '/foo/npm/npmjs/-/test/revision/0.0',
-        '/foo/npm/npmjs/-/test/revision/1.0/tool/testTool/2.0.json',
-        '/foo/npm/npmjs/-/test/revision/2.0/tool/testTool0/1.0.json',
-        '/foo/npm/npmjs/-/test/revision/2.0/tool/testTool1/2.0.json',
-        '/foo/npm/npmjs/-/test/revision/2.0/tool/testTool2/3.0.json'
+        '/foo/npm/npmjs/-/test/revision/1.0/tool/testtool/2.0.json',
+        '/foo/npm/npmjs/-/test/revision/2.0/tool/testtool0/1.0.json',
+        '/foo/npm/npmjs/-/test/revision/2.0/tool/testtool1/2.0.json',
+        '/foo/npm/npmjs/-/test/revision/2.0/tool/testtool2/3.0.json'
```

Comment:
this is the stubbed result of recursive-dir - all the files are lowercased as they are stored now, so updated the stub to return lowercased files